styles.css

```diff
@@ -0,0 +1,32 @@
+body {
+  padding-top: 70px;
+}
+
+h1 {
+  color:tomato;
+}
+
+.notShown {
+  display: none;
+}
+
+.visible {
+  display: block;
+}
```

Comment:
Not vital, but you could probably refrain from having both a notShown and a visible class, since one is really just the absence of the other...